src/pages/Main/Tealist/Detail/Detail.jsx

```diff
@@ -1,10 +1,130 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+import Sharebutton from './Sharebutton/Sharebutton';
+import Selectlist from './Selectlist/Selectlist';
+import './Detail.scss';
 
 class Detail extends React.Component {
+  state = {
+    data: [],
+    count: 0,
+    bagPrice: 0,
+  };
+
+  componentDidMount() {
+    window.Kakao.init('8b745f5f2ce644953d70506049de4e81');
+
+    window.Kakao.Link.createDefaultButton({
+      container: '#kakao-link-btn',
+      objectType: 'feed',
+      content: {
+        title: '프리미엄 말차',
+        description:
+          '즐겁고 행복한 티타임을 선사하는 달콤하고 향긋한 오설록만의 특별한 블렌디드 티 선물세트입니다.',
+        imageUrl:
+          'https://www.osulloc.com/upload/kr/ko/adminImage/PW/US/20180406140800938CQ.png?quality=80',
+        link: {
+          webUrl: 'http://localhost:3000/detail',
+        },
+      },
+      buttons: [
+        {
+          title: '녹차록에서 확인하기',
+          link: {
+            mobileWebUrl: 'http://localhost:3000/detail',
+            webUrl: 'http://localhost:3000/detail',
+          },
+        },
+      ],
+    });
+  }
+
+  includeBagPrice = enclose => {
+    enclose
+      ? this.setState(() => ({ bagPrice: 100 }))
+      : this.setState(prev => ({ bagPrice: 0 }));
```

Comment:
```suggestion
      ? this.setState({ bagPrice: 100 })
      : this.setState({ bagPrice: 0 });
```
이전 state를 활용하는 부분이 없는 것 같습니다 이 경우엔 굳이 함수형태의 인자를 전달할 필요가 없겠네요!

Reply:
넵! 수정했습니다!